app/routes.js

```diff
@@ -1,15 +1,17 @@
 import React from 'react';
 import { Route, IndexRoute } from 'react-router';
-
-import RootContainer from './containers/RootContainer';
+import RootContainer from './containers/connectors/RootContainerConnector';
 import About from './components/About';
-import IndexMain from './containers/IndexMain';
+import IndexMain from './containers/connectors/IndexMainConnector';
+import WrongPath from './components/WrongPath';
 
-const router = (
+const routes = (
     <Route path="/" component={RootContainer}>
-        <Route path="/about" component={About} />
-        <Route path="/home" component={IndexMain} />
         <IndexRoute component={IndexMain} />
+        <Route path="/home" component={IndexMain} />
+        <Route path="/cities/:country/:cityname" component={IndexMain} />
+        <Route path="/about" component={About} />
```

Comment:
@AlexeyBarkow if you go to about page and then reload, background image will be empty.